src/ape_init/_cli.py

```diff
@@ -9,8 +9,26 @@
 
 @click.command(short_help="Initalize an ape project")
 @ape_cli_context()
-@click.option("--github")
+@click.option(
+    "--github", metavar="GH_username/repo", help="Add github_username/repository to clone project"
+)
 def cli(cli_ctx, github):
+    """
+    Ape Init allows the user to create an ape project with
```

Comment:
```suggestion
    ``ape init`` allows the user to create an ape project with
```

Although, click doesn't render the backticks by default... Wonder if there is a way to do this